test/e2e/util/util.go

```diff
@@ -426,9 +426,9 @@ func EnsureAPIBudget(t *testing.T, ctx context.Context, client crclient.Client,
 				}
 				for _, sample := range vector {
 					if float64(sample.Value) > budget.budget {
-						t.Errorf("over budget: budget: %.0f, actual: %.0f", budget.budget, sample.Value)
+						t.Errorf("%q over budget: budget: %.0f, actual: %.0f", budget.name, budget.budget, sample.Value)
```

Comment:
The `budget.Name` is already in the testcase name (`controlplane operator read` in the sample PR that failed), so what we need here is some more details about the `sample` I think